DirectEmployers/MyJobs
# [PD 3155] make category required

## What's this PR do?
This PR makes the categories a required field when adding a contact inside of CBO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull down the repository. You can either go to a CBO and add an additional contact from the contact list or you can create a new cbo and add a contact that way as well. If you try and go to the next page or save without selecting a contact you will see the validation message popup prompting that a category is going to be required

## Any background context you want to provide?
